prefect_census/credentials.py

```diff
@@ -0,0 +1,68 @@
+"""Module containing credentials for interacting with Census."""
+from httpx import AsyncClient
+from prefect.blocks.core import Block
+from pydantic import SecretStr
+
+from prefect_census.client import CensusClient
+
+
+class CensusCredentials(Block):
+    """
+    Credentials block for credential use across Census tasks and flows.
+
+    Attributes:
+        api_key (SecretStr): API key to authenticate with the Census
+        API. Refer to the [Authentication docs](
+        https://docs.getcensus.com/basics/api#getting-api-access)
+        for retrieving the API key.
```

Comment:
No types needed in the Attributes section; also indent for following lines.

```suggestion
        api_key: API key to authenticate with the Census
            API. Refer to the [Authentication docs](
            https://docs.getcensus.com/basics/api#getting-api-access)
            for retrieving the API key.
```